asset/js/widget/SearchBar.js

```diff
@@ -39,9 +39,28 @@ define(["../notjQuery"], function ($) {
             let opener = event.currentTarget;
             let editorUrl = opener.dataset.searchEditorUrl;
             let filterQueryString = this.filterInput.getQueryString();
+            let layout = document.getElementById('layout');
 
             editorUrl += (editorUrl.indexOf('?') > -1 ? '&' : '?') + filterQueryString;
 
+            // Disable pointer events to block further function calls
+            setTimeout(function () {
+                opener.style.pointerEvents = 'none';
+            }, 20);
```

Comment:
Why not instantly? Or why after 20ms?

Reply:
I have not been able to find out why, but it does not work instantly.

Reply:
Then please try `0` for the delay. This way it's right next in the event flow, asap so to say.

Reply:
Tried without `setTimeOut` and works fine now 🤦🏽 (mysterious behaviour).